content/Teams/observability-platform/sre.md

```diff
@@ -0,0 +1,130 @@
+
+Currently, we offer internal service called [RHOBS](../../Projects/Observability/RHOBS) that is used across the company.
+
+## Releasing Changes / Patches
+
+In order to maintain our software on production reliably we need to be able to test, release and deploy our changes rapidly.
+
+We can divide things we change into a few categories. Let's elaborate all processes per each category:
+
+### Software Libraries
+
+Library is a project that is not deployed directly, but rather it is a dependency for [micro-service application](#software-for-micro-services) we deploy. For example https://github.com/prometheus/client_golang or https://github.com/grpc-ecosystem/go-grpc-middleware
+
+*Testing*:
+* Linters, unit and integration tests on every PR.
+
+*Releasing*
+* GitHub release using git tag (RC first, then major/minor or patch release).
+
+*Deploying*
+* Bumping dependency in Service
+* Following on Service [release and deploy steps](#software-for-micro-services).
+
+### Software for (Micro) Services
+
+Software for (micro) services usually lives in many separate open source repositories in GitHub e.g https://github.com/thanos-io/thanos, https://github.com/observatorium/api.
+
+*Testing*:
+* Linters, unit and integration tests on every PR.
+
+*Releasing*
+* GitHub release using git tag (RC first, then major/minor or patch release).
+* Building docker images in quay.io and dockerhub (backup) using CI for each tag or main branch PR.
+
+*Deploying*
+* Bumping docker tag version in configuration.
+* Following [configuration release and deployment steps](#configuration).
+
+### Configuration
+
+All or configuration is rooted in https://github.com/rhobs/configuration configuration templates written in jsonnet. It can be then overridden by defined parameters in app-interface saas files  .
+
+*Testing*:
+* [Building jsonnet resources](https://github.com/rhobs/configuration/blob/main/.github/workflows/ci.yml), linting jsonnet, validating openshift templates.
+* Validation on App-interface side against Kubernetes YAMLs compatibility
+
+*Releasing*
+* Merge to main and get the commit hash you want to deploy.
+
+*Deploying*
+* Make sure you are on RH VPN.
+* Propose PR to https://gitlab.cee.redhat.com/service/app-interface with the change of ref for the desired environment to desired commit sha is app-interface Saas file for desired tenant ([telemeter](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/rhobs/telemeter/cicd/saas.yaml#L70) or MST) or change environment parameter.
+* Ask team for review. If change is impacting production heavily, notify AppSRE.
+  * If only `saas.yaml` file was changed `/lgtm` from Observability Platform team is enough for PR to get merged automatically.
+  * If any other file was changed, AppSRE engineer has to lgtm it.
+* When merged, CI will deploy the changes to cluster specified in `saas.yaml` e.g to production.
```

Comment:
```suggestion
* When merged, CI will deploy the changes to namespace specified in `saas.yaml` e.g to production.
```

`saas` files only deals with namespace level targets, the cluster on which the namespace lives is part of the namespace definition.